package.json

```diff
@@ -49,6 +49,7 @@
     "cross-env": "^5.0.5",
     "eslint": "^4.0.0",
     "esprima": "^4.0.0",
+    "glob": "^7.1.2",
```

Comment:
Why's this?

Reply:
@aslushnikov -  Hi, thanks for replying. 

I'm not sure I explained my solution well: I changed `test/test.js` to setup the runner, find all files ending with `.spec.js` (reason for adding `glob` as a dev dependency), and then require them. When those files are required they register new tests for the runner by calling the globally available `describe`/`it` functions. Once they're all `require`ed, we call `runner.run()`.

This results in all tests running in the same node process, exactly like they currently do, just that now they're split over multiple files.

With this change, `test/test.js` is the only runnable file. We can still run just one test with `fit`.

I think that splitting those two files even further should be easy now :)

Reply:
please see my explanation [here](https://github.com/GoogleChrome/puppeteer/pull/2193#issuecomment-373443452)